Controls/Chart/Chart.UWP/Visualization/CartesianChart/CartesianChartGrid.cs

```diff
@@ -532,11 +569,20 @@ private int ArrangeLines()
                         AxisTickModel nextTick = stripe.AssociatedTick.NextMajorTick;
                         if (nextTick.isVisible)
                         {
-                            line = this.GetLineVisual(lineCount);
                             RadRect lastRect = this.Orientation == Windows.UI.Xaml.Controls.Orientation.Vertical ?
-                                new RadRect(stripe.BorderRect.Right, stripe.BorderRect.Y, 1, stripe.BorderRect.Height) :
-                                new RadRect(stripe.BorderRect.X, stripe.BorderRect.Y - 1, stripe.BorderRect.Width, 1);
-                            this.ArrangeLine(lastRect, line);
+                                    new RadRect(stripe.BorderRect.Right, stripe.BorderRect.Y, 1, stripe.BorderRect.Height) :
```

Comment:
Yes we can. I believe this the width/height of the Line visualized in the ChartGrid.